src/lib/SCEditor.js

```diff
@@ -379,6 +380,18 @@ export default function SCEditor(original, userOptions) {
 	// Don't deep extend emoticons (fixes #565)
 	base.opts.emoticons = userOptions.emoticons || defaultOptions.emoticons;
 
+	/**
+	 * Sanitize HTML to avoid XSS
+	 *
+	 * @param {string} html
+	 * @return {string} html
+	 */
+	base.sanitize = function (html) {
```

Comment:
That would be a good to do if it remains.

While testing this I've realised it might not need to be exposed. If plugins are only ever passed sanitised HTML and all HTML received from plugins is sanitised then only possible reason for a plugin to need to use this would be if it accepted HTML user input directly and inserted it into the DOM directly.

None of the default plugins would need this to be exposed. For now it might make more sense to make this private and expose it later if a use case is found.